### PR TITLE
feat(rdb): encryption at rest api + bench MTA-5369

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: managed-databases/postgresql-and-mysql/reference-content/encryption-at-rest-performance-benchmark.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+tags: managed-database postgresql mysql encryption at rest
+dates:
+  validation: 2024-12-19
+  posted: 2024-12-19
+categories:
+  - managed-databases
+  - postgresql-and-mysql
+---
+
+The Scaleway Managed Databases team ran tests on DB-POP2-4C-16G Instances in a Private Network using `pgbench`. Several workloads (including read, oltp and large loads) were simulated with several scale factors (from 1M rows to 200M rows) and concurrent database connections set to 10. They measured the impact of encryption at rest on transactions, per second and latency.
```
Comment (scw-wtimmermans) [inline managed-databases/postgresql-and-mysql/reference-content/encryption-at-rest-performance-benchmark.mdx:17]: We can add here that these are maximal numbers and that no one during the Public Beta complained about any Performances issues. If they perform their own benchmarks, we are eager to see their figures and they should not hesitate to share them on our slack community channel

File: managed-databases/postgresql-and-mysql/reference-content/encryption-at-rest-performance-benchmark.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+<Message type="important">
+  For more information on pgbench transactions, refer to the official [pgbench documentation](https://www.postgresql.org/docs/current/pgbench.html).
```
[the remark this answers is not in the document]
Comment (ldecarvalho-doc) [reply managed-databases/postgresql-and-mysql/reference-content/encryption-at-rest-performance-benchmark.mdx:20]:
```suggestion
  For more information on how transactions performed in pgbench are defined, refer to the official [pgbench documentation](https://www.postgresql.org/docs/current/pgbench.html).
```

File: managed-databases/postgresql-and-mysql/api-cli/setting-up-encryption-at-rest.mdx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    <Message type="note">
+      In all examples below the UUIDs, passwords and IP addresses displayed are not real. For more details about the parameters shown, refer to the [Managed Databases for PostgreSQL and MySQL API documentation](https://www.scaleway.com/en/developers/api/managed-database-postgre-mysql/)
+    </Message>
+
+    ```json
+      '{
+      "project_id": "d8e65f2b-cce9-40b7-80fc-6a2902db6826",
+      "name": "myDB",
+      "engine": "PostgreSQL-15",
+      "tags": ["donnerstag"],
```
Comment (Uyoooo) [inline managed-databases/postgresql-and-mysql/api-cli/setting-up-encryption-at-rest.mdx:39]: May be a more abtract tag here like ["foo", "bar],

**Review by RoRoJ — APPROVED** [review]

File: managed-databases/postgresql-and-mysql/reference-content/encryption-at-rest-performance-benchmark.mdx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+<Message type="important">
+  For more information on pgbench transactions, refer to the "What is the 'Transaction' actually performed in pgbench?" section of the official [pgbench documentation](https://www.postgresql.org/docs/current/pgbench.html).
+</Message>
+
+<Message type="note">
+  The workloads applied simulate heavy usage scenarios for the purposes of testing. During the beta of encryption at rest, no performance issues were raised by users. If you would like to share your own benchmarks with our team, feel free to reach out to us in the #database channel of the [Scaleway Slack community](/tutorials/scaleway-slack-community/).
+</Message>
+
+The table below compares the different workload and setup combinations and indicates the perfomance results of each:
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/reference-content/encryption-at-rest-performance-benchmark.mdx:27]:
```suggestion
The table below compares the different workload and setup combinations and indicates the performance results of each:
```

File: managed-databases/postgresql-and-mysql/reference-content/encryption-at-rest-performance-benchmark.mdx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+tags: managed-database postgresql mysql encryption at rest
+dates:
+  validation: 2024-12-19
+  posted: 2024-12-19
+categories:
+  - managed-databases
+  - postgresql-and-mysql
+---
+
+In December 2024, he Scaleway Managed Databases team ran tests on DB-POP2-4C-16G Instances in a Private Network using `pgbench`. Several workloads (including read, oltp and large loads) were simulated with several scale factors (from 1M rows to 200M rows) and concurrent database connections set to 10. They measured the impact of encryption at rest on transactions, per second and latency.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/reference-content/encryption-at-rest-performance-benchmark.mdx:17]:
```suggestion
In December 2024, the Scaleway Managed Databases team ran tests on DB-POP2-4C-16G Instances in a Private Network using `pgbench`. Several workloads (including read, oltp and large loads) were simulated with several scale factors (from 1M rows to 200M rows) and concurrent database connections set to 10. They measured the impact of encryption at rest on transactions, per second and latency.
```

File: managed-databases/postgresql-and-mysql/api-cli/setting-up-encryption-at-rest.mdx [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+      "password": "thiZ_is_v0ry_s3cret",
+      "volume_type": "sbs_5k",
+      "volume_size": "30000000000",
+      "encryption": {
+        "enabled": true
+        }
+      }'
+    ```
+
+3. Create a Database Instance by running the following command. Make sure you include the payload you edited in the previous step.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/api-cli/setting-up-encryption-at-rest.mdx:53]:
```suggestion
2. Create a Database Instance by running the following command. Make sure you include the payload you edited in the previous step.
```

**Review by jcirinosclwy — APPROVED** [review]

**Pull request merged**